genie.py

```diff
@@ -92,22 +94,29 @@ def __init__(
             use_flash_attention=self.use_flash_attention,
             rngs=rngs,
         )
-        self.lam = LatentActionModel(
-            in_dim=self.in_dim,
-            model_dim=self.lam_dim,
-            ffn_dim=self.lam_ffn_dim,
-            latent_dim=self.latent_patch_dim,
-            num_latents=self.num_latent_actions,
-            patch_size=self.lam_patch_size,
-            num_blocks=self.lam_num_blocks,
-            num_heads=self.lam_num_heads,
-            dropout=0.0,
-            codebook_dropout=0.0,
-            param_dtype=self.param_dtype,
-            dtype=self.dtype,
-            use_flash_attention=self.use_flash_attention,
-            rngs=rngs,
-        )
+        if self.use_gt_actions:
+            self.action_embed = nnx.Embed(
+                self.latent_action_dim, self.latent_action_dim, rngs=rngs
```

Comment:
The first argument should be `num_latent_actions`.
```suggestion
                self.num_latent_actions, self.latent_action_dim, rngs=rngs
```